[BUG] Fix TSError: Unable to compile TypeScript

## summary
TSError occurs during mocha integral test.

`error TS6133: 'OpCode' is declared but its value is never read.`
`error TS2552: Cannot find name 'Opcode'.`
## 관련 Endpoint

## 버그 재현 경로

## 필요한 작업

- [ ] Replace incorrectly declared variable names with correct variable names

## 스크린샷
<img width="1148" alt="스크린샷 2023-01-03 오후 12 59 42" src="https://user-images.githubusercontent.com/35447878/210299037-1edec569-e1ad-440d-8869-19bb30fe42b6.png">

--- a/tests/test_integral/auth.spec.ts
+++ b/tests/test_integral/auth.spec.ts
@@ -11,7 +11,7 @@ import app from '@src/app';
 import { TokenSetDTO } from '@dto/index';
 import { BASE_PATH, ABNORMAL_CONNECTION } from '@utils/strings';
 
-import { ResponseDTO, SimpleResponseDTO, Opcode } from '@response/common';
+import { ResponseDTO, SimpleResponseDTO, OpCode } from '@response/common';
 import { LoginResponse } from '@response/user';
 
 const basePath: string = BASE_PATH;
@@ -122,7 +122,7 @@ describe('# Auth Controller Test', () => {
                                                 responseDTO.message
                                             ).to.be.eq(ABNORMAL_CONNECTION);
                                             expect(responseDTO.opcode).to.be.eq(
-                                                Opcode.LOGOUT
+                                                OpCode.LOGOUT
                                             );
                                             done();
                                         }

